grails-app/conf/Config.groovy

```diff
@@ -93,6 +93,10 @@ googleAnalytics.trackingId = null
 // set per-environment serverURL stem for creating absolute links
 def env = System.getenv()
 
+filtering {
+    loadFromFiles = Boolean.parseBoolean(env['LOAD_FILTERS_FROM_FILES'])
```

Comment:
I'd say if you're already working on that - have it as a string which represents a path rather than a true/false toggle. This will be useful for IMAS for instance.